stdlib/float.ml

```diff
@@ -85,6 +85,8 @@ external modf : float -> float * float = "caml_modf_float"
 type t = float
 external compare : float -> float -> int = "%compare"
 let equal x y = compare x y = 0
+external seeded_hash_param : int -> int -> int -> float -> int = "caml_hash" [@@noalloc]
+let hash x = seeded_hash_param 10 100 0 x
```

Comment:
(Later, it could be useful to create an ad hoc runtime primitive for Float.hash, with an unboxed form.)